### PR TITLE
138 disable update to fiware service for project admins

### LINKED ISSUE #138: disable update to fiware service for project admins

### PR DESCRIPTION
**Pull request by sbanoeon**

### CODE REVIEW

**Comment by djs0109** [comment]
@sbanoeon  I found that users created by admin sole cannot login to entirety. I suppose there is something wrong with the saving of password. Could you also confirm that?

**Comment by sbanoeon** [comment]
@djs0109 I can login local users (which are created through django create user command line). Do you mean the same?

**Comment by djs0109** [comment]
@sbanoeon I made commit to fix that, can you have a look? The other codes lgtm :) After you give your consent to that commit, I will approve the PR

**Review by djs0109 — APPROVED** [review]

**Pull request merged**
